README.md

```diff
@@ -215,6 +215,15 @@ The descriptions of these items:
  * `-nts` - optional - non-thread safe. If omitted, thread-safe mode (ZTS) is assumed.
  * `-arch` - for example `x86_64`, fetch using `php -r "echo php_uname('m');"`.
 
+#### Contents of the Windows ZIP
+
+The pre-built ZIP should contain at minimum a DLL named `php_{extension-name}.dll`. It may include additional resources,
```

Comment:
I would really like to see `php_{extension-name}-{tag}-{php-maj/min}-{compiler}{"-nts"}?{-arch}.dll` as filenames. and not just `php_{extension-name}.dll`, as having multiple `php_{extension-name}.dll` files makes it really hard to see which is which.

Reply:
This kinda goes against the convention of how the .dlls are already named, I believe? Once they're in situ, the php version is usually going to be inferred by the path, or maybe there is only one PHP path on the system, e.g. `C:\php\php8.1.2\ext\php_xdebug.dll` - it's not a dealbreaker to change this to use the "detailed name" as you suggested, but I wonder if it's superfluous? :thinking:

Reply:
I was only talking about the contents of the zip file. I agree they should be *installed* as `php_{extension_name}.dll`.

Reply:
Understood, yes I think that's acceptable :+1:

Reply:
Fixed in 1d94de9762a5145dc47ea97385355cb7dc0c186e - please see that update @shivammathur for when the package is produced, the included `.dll` naming should match the `.zip`

Reply:
Noted 👍. I will make the change